docs/reference/ingest/search-ingest-pipelines.asciidoc

```diff
@@ -0,0 +1,313 @@
+You can view these pipelines using the platform tools (Kibana UI, Elasticsearch API), and can also 
+//<<ingest-pipeline-search-details-specific,read more about their content below>>.
+
+[discrete#ingest-pipeline-search-pipeline-settings]
+=== Pipeline Settings
+
+Aside from the pipeline itself, *Search* provides a few configuration options which control individual features of the pipelines.
+
+* **Extract Binary Content** - This controls whether or not binary documents should be processed and any textual content should be extracted.
+* **Reduce Whitespace** - This controls whether or not consecutive, leading, and trailing whitespaces should be removed.
+  This can help to display more content in some search experiences.
+* **Run ML Inference** - Only available on index-specific pipelines.
+  This controls whether or not the optional `<index-name>@ml-inference` pipeline will be run.
+  Enabled by default.
+
+For Elastic web crawler and connectors, you can opt in or out per index.
+These settings are stored in Elasticsearch in the `.elastic-connectors` index, in the document that corresponds to the specific index.
+These settings can be changed there directly, or through the Kibana UI at *Search > Content > Indices > <your index> > Pipelines > Settings*.
+
+You can also change the deployment wide defaults.
+These settings are stored in the Elasticsearch mapping for `.elastic-connectors` in the `_meta` section.
+These settings can be changed there directly, or from the Kibana UI at *Search > Content > Settings* tab.
+Changing the deployment wide defaults will not impact any existing indices, but will only impact any newly created indices defaults.
+Those defaults will still be able to be overriden by the index-specific settings.
+
+[discrete#ingest-pipeline-search-pipeline-settings-using-the-api]
+==== Using the API
+
+These settings are not persisted for indices that "Use the API".
+Instead, changing these settings will, in real time, change the example cURL request displayed.
+Notice that the example document in the cURL request contains three underscore-prefixed fields:
+
+[source,js]
+----
+{
+  ...
+  "_extract_binary_content": true,
+  "_reduce_whitespace": true,
+  "_run_ml_inference": true
+}
+----
+// NOTCONSOLE
+
+Omitting one of these special fields is the same as specifying it with the value `false`.
+
+[NOTE]
+=========================
+You must also specify the pipeline in your indexing request.
+This is also shown in the example cURL request.
+=========================
+
+[WARNING]
+=========================
+If the pipeline is not specified, the underscore-prefixed fields will actually be indexed, and will not impact any processing behaviors.
+=========================
+
+[discrete#ingest-pipeline-search-details]
+=== Details
+
+[discrete#ingest-pipeline-search-details-generic-reference]
+==== `ent-search-generic-ingestion` Reference
+
+As soon as *Search* starts, this pipeline is automatically available to use.
```

Comment:
This sentence no longer relevant, now that Enterprise Search is no longer a dependency.